index.html

```diff
@@ -107,6 +107,74 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long numFreezes;
+      unsigned long numPauses;
+      double totalFreezesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>numFreezes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video freezes experienced by this receiver. It is a
+          freeze if frame duration, which is time interval between consecutively
+          rendered frames, is 3 times higher than the average frame duration.
```

Comment:
How is the average calculated? Is it average over the last x seconds, average between all received frames, etc.

Reply:
Good Q. It is avg between all. 
Usage - https://cs.chromium.org/chromium/src/third_party/webrtc/video/video_quality_observer.cc?rcl=71b5a7df7794bbc4103296fcd8bd740acebdc901&l=152

Implementation - https://cs.chromium.org/chromium/src/third_party/webrtc/rtc_base/numerics/sample_counter.cc?rcl=71b5a7df7794bbc4103296fcd8bd740acebdc901&l=53

This doesn't look right. I will replace it with moving avg (5 secs window or something, need to decide) and update the description.

Reply:
Replaced with MA of durations of last 30 rendered frame in https://webrtc-review.googlesource.com/c/src/+/114840/13.

Reply:
The description has been updated as well.